### PR TITLE
fix #7619: don't wrap menu action args into another array

### LINKED ISSUE #7619: more command (`...`) in view containers does not work anymore
It is a regression.

### PR DESCRIPTION
**Pull request by akosyakov**
#### What it does

- add integration tests to reproduce fix #7619
- fix #7619 by removing bogus spreading

#### How to test

- integration tests should be green
- test that context menus are properly rendered in different places, i.e. toolbars for view container, views, editors and so on

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/core/src/browser/shell/side-panel-toolbar.ts [hunk 1]
```diff
@@ -101,4 +102,13 @@ export class SidePanelToolbar extends BaseWidget {
             this.update();
         }
     }
+
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    showMoreContextMenu(anchor: Anchor): ContextMenuAccess {
```
Comment (AlexTugarev) [inline packages/core/src/browser/shell/side-panel-toolbar.ts:107]: is this for testing purposes only?

Comment (akosyakov) [reply packages/core/src/browser/shell/side-panel-toolbar.ts:107]: for now yes

Comment (AlexTugarev) [reply packages/core/src/browser/shell/side-panel-toolbar.ts:107]: it's a side effect, though.
have you considered special marking if the test is the single client?

Comment (akosyakov) [reply packages/core/src/browser/shell/side-panel-toolbar.ts:107]: I'm not sure what you mean.

**Pull request merged**